core/src/main/resources/META-INF/patches/sql/postgresql/01-create-jahia-external-provider-schema.resolved.sql

```diff
@@ -18,3 +18,5 @@
     create index jahia_external_provider_id_ix1 on jahia_external_provider_id (providerKey);
 
     create sequence jahia_external_provider_id_seq;
+
+    CREATE TRIGGER t_oid_jahia_external_mapping_externalid BEFORE DELETE OR UPDATE ON jahia_external_mapping FOR EACH ROW EXECUTE FUNCTION lo_manage('externalid');
```

Comment:
should it be **externalId** instead of **externalid**?

Reply:
In the internal schema it's "internalid" (this trigger was generated automatically from the schema). Maybe postgres is lowercasing automatically all table/column names  ?